index_twitter_stream.py

```diff
@@ -17,13 +17,20 @@
 from esconn import esconn
 es = esconn()
 
-# Update with topics to follow
-if sys.argv[1]:
-    f = open(sys.argv[1])
+if len(sys.argv) == 2:
```

Comment:
I agree argparse or other CLI module (or a config file) is better if it reaches a point we are passing a lot of arguments.  For 1 optional argument seems we can keep it simple for now. I originally had >= 2 but I landed on this because I felt it was more explicit on exactly what the purpose of checking length was. If we want to handle all possible input I would say it's more clean to reject anything with more than 1 additional argument. Checked in that proposal.